pyaerocom/web/aerocom_evaluation.py

```diff
@@ -885,6 +888,22 @@ def find_obs_matches(self, name_or_pattern):
                            .format(name_or_pattern))
         return matches
 
+    def _check_and_get_iface_names(self):
+        obs_list = list(self.obs_config)
+        iface_names = []
+        for obs_name in obs_list:
+            try:
+                if self.obs_config[obs_name]['web_interface_name'] == None:
+                    self.obs_config[obs_name]['web_interface_name'] = obs_name
+                else:
+                    pass
+            except KeyError:
+                self.obs_config[obs_name]['web_interface_name'] = obs_name
+            assert isinstance(self.obs_config[obs_name]['web_interface_name'],str)
+            iface_names.append(self.obs_config[obs_name]['web_interface_name'])
+        iface_names = set(iface_names)
```

Comment:
Just curious: why making this a set?

Reply:
I wanted to make sure there were no duplicates in the list. I'm not sure it's necessary for the way I've implemented this, but it seemed necessary at the time.